acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -126,25 +158,34 @@ private void pullPantheonImage() {
   }
 
   private String createPantheonContainer(final NodeConfiguration config) {
-    final HostConfig portBindingConfig =
-        HostConfig.newHostConfig().withPortBindings(tcpPortBinding(config), wsPortBinding(config));
+    @SuppressWarnings("unstable")
+    final String genesis = Resources.getResource(config.getGenesisFile()).getPath();
+    LOG.info("Path to Genesis file: {}", genesis);
+    final Volume genesisVolume = new Volume("/etc/pantheon/genesis.json");
```

Comment:
is the volume stuff safe to use without the jenkins fix? or has the been fixed too?

Reply:
? what do you mean?

Reply:
Yes, it works on Jenkins

Reply:
yeah I meant binding the genesis file. but looks like Jenkins issue is all fixed. so ignore my comment 😁